securityscanutils/securityscan.go

```diff
@@ -137,7 +142,7 @@ func (s *SecurityScanner) GenerateSecurityScans(ctx context.Context) error {
 
 			// Only generate sarif files if we are uploading code scan results to github
 			if repo.Opts.UploadCodeScanToGithub {
-				err = repo.RunGithubSarifScan(release, sarifTplFile)
+				err = repo.RunGithubSarifScan(ctx, release, sarifTplFile)
```

Comment:
Since this is an exported function, this is technically a breaking change. You could argue that this function should never have been exported in the first place as its intended to be called internally. i'll leave it up to you whether you want to make this a minor version bump as a result, or just acll it out loudly in the PR.

As other teams pull in these changes, we need to make it easy to identify how to upgrade to this version and what the impacts are

Reply:
Might want to do a minor version bump just to keep consistent

Reply:
Any functions in particular we should make private other than this one?